machinelearning/tensorflow.md

```diff
@@ -217,7 +208,8 @@ OPENMP DISPLAY ENVIRONMENT END
 
 # Building TensorFlow from sources
 
-While the packages for python wheel/docker container/DLAMI provide stable baseline for ML application development and production, they lack the latest fixes and optimizations from the development branches. This section provides instructions for building TensorFlow from sources, to build the master branch or to incorporate the downstream optimizations.
+With AWS Graviton DLCs and python wheels available for every official TensorFlow release, there may not be a need for customers to build TensorFlow from sources. However, to make the user guide complete, this section provides instructions for building the wheels from sources.
```

Comment:
What is special about building for Graviton that we would still need this now that all the AMIs, containers and wheels have full Arm support?

Reply:
I have update this section to set the scope for source builds. As you said, there is no need for source builds and we recommended only the official DLCs. But I have provided the instructions for developers who would like to experiment their custom features on aarch64.